src/database/guild.service.js

```diff
@@ -0,0 +1,20 @@
+import {GuildModel} from "./models/guild.js";
+
+export class GuildService {
```

Comment:
```suggestion
export class DatabaseService {
```
Also rename the file to `database.service.js` so we can have all database functions in one place?